### ISSUE TITLE
Verification Email

### ISSUE THREAD

**Issue by mcdappdev** [atech/postal#75]
I am trying to setup a Postal server and I'm not getting the initial verification email despite resending it multiple times. Is this a known issue?

**Comment by anhofmann** [comment]
This was a problem for me to. The problem is that while in setup postal has no clue how to send mails. I had to look in the users table of the postal database for the verification code:
`mysql -p`
`select email_address, email_verification_token from postal.users;`

**Comment by mcdappdev** [comment]
That's what I ended up doing too! Thanks :)

**Comment by adamcooke** [comment]
You can test your app SMTP settings by running `postal test-app-smtp [email]`. This will attempt to send an a message to the given address in the same way as the application sends verification e-mails. If an error occurs, you'll be told what it is.

**Comment by anhofmann** [comment]
I did this and other things as described below. This is a hen-or-egg-problem. I started with a fresh ubuntu 16.04 system that had no clue how to send mails to the internet. I started to walk through the steps of the installation process and jumped the step for smtp config in the postal.yml so postal was not able to send the confirmation mail. The installation page recommend to do this: 

> For now, you can leave this as it is configured but no internal messages will be sent. Once you have your Postal installation up and running you can come back to this and add details so it can send messages through itself.

I have used postfix since decades so I decided to install postfix and put localhost in the smtp config. But Postal tried to connect with SSL and this didn't worked out of the box. After that I copied my activation key out of the database. So if someone wants to install Postal on a fresh system, there seems to be no other way to get the first Postal Account activated.
